spartan/terraform/deploy-rollup-contracts/main.tf

```diff
@@ -35,30 +35,35 @@ locals {
 
 
   # Environment variables for the container (omit keys with null values)
-  env_vars = { for k, v in {
-    AZTEC_SLOT_DURATION                      = var.AZTEC_SLOT_DURATION
-    AZTEC_EPOCH_DURATION                     = var.AZTEC_EPOCH_DURATION
-    AZTEC_TARGET_COMMITTEE_SIZE              = var.AZTEC_TARGET_COMMITTEE_SIZE
-    AZTEC_PROOF_SUBMISSION_EPOCHS            = var.AZTEC_PROOF_SUBMISSION_EPOCHS
-    AZTEC_ACTIVATION_THRESHOLD               = var.AZTEC_ACTIVATION_THRESHOLD
-    AZTEC_EJECTION_THRESHOLD                 = var.AZTEC_EJECTION_THRESHOLD
-    AZTEC_SLASHING_QUORUM                    = var.AZTEC_SLASHING_QUORUM
-    AZTEC_SLASHING_ROUND_SIZE                = var.AZTEC_SLASHING_ROUND_SIZE
-    AZTEC_SLASHING_ROUND_SIZE_IN_EPOCHS      = var.AZTEC_SLASHING_ROUND_SIZE_IN_EPOCHS
-    AZTEC_SLASHING_LIFETIME_IN_ROUNDS        = var.AZTEC_SLASHING_LIFETIME_IN_ROUNDS
-    AZTEC_SLASHING_EXECUTION_DELAY_IN_ROUNDS = var.AZTEC_SLASHING_EXECUTION_DELAY_IN_ROUNDS
-    AZTEC_SLASHING_VETOER                    = var.AZTEC_SLASHING_VETOER
-    AZTEC_SLASHING_OFFSET_IN_ROUNDS          = var.AZTEC_SLASHING_OFFSET_IN_ROUNDS
-    AZTEC_SLASH_AMOUNT_SMALL                 = var.AZTEC_SLASH_AMOUNT_SMALL
-    AZTEC_SLASH_AMOUNT_MEDIUM                = var.AZTEC_SLASH_AMOUNT_MEDIUM
-    AZTEC_SLASH_AMOUNT_LARGE                 = var.AZTEC_SLASH_AMOUNT_LARGE
-    AZTEC_SLASHER_FLAVOR                     = var.AZTEC_SLASHER_FLAVOR
-    AZTEC_GOVERNANCE_PROPOSER_QUORUM         = var.AZTEC_GOVERNANCE_PROPOSER_QUORUM
-    AZTEC_GOVERNANCE_PROPOSER_ROUND_SIZE     = var.AZTEC_GOVERNANCE_PROPOSER_ROUND_SIZE
-    AZTEC_MANA_TARGET                        = var.AZTEC_MANA_TARGET
-    AZTEC_PROVING_COST_PER_MANA              = var.AZTEC_PROVING_COST_PER_MANA
-    AZTEC_EXIT_DELAY_SECONDS                 = var.AZTEC_EXIT_DELAY_SECONDS
-    LOG_LEVEL                                = "debug"
+  # if NETWORK is set, ignore the other env vars
+  env_vars = var.NETWORK != null ? {
+    NETWORK         = var.NETWORK
+    LOG_LEVEL       = "debug"
+    BOOTSTRAP_NODES = "asdf"
```

Comment:
ugh is this because passing NETWORK=foo automatically tries to load the bootnodes (if BOOTSTRAP_NODES is missing) and if the file is not there is fails?

I created TMNT-305 so we can go back and clean this up in the future.